M/MuJoCo/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "MuJoCo"
+version = v"2.3.7"
+
+# Collection of sources required to build mujoco
+sources = [
+    GitSource("https://github.com/deepmind/mujoco", "c9246e1f5006379d599e0bcddf159a8616d31441"),
+    DirectorySource(joinpath(@__DIR__, "patches"))
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mujoco
+if [[ "${target}" == *-mingw* ]]; then
+    atomic_patch -p1 ../mingw.patch
+elif [[ "${target}" == *-apple-darwin* ]]; then
+    atomic_patch -p1 ../macos.patch
+else
+    atomic_patch -p1 ../other.patch
+fi
+CPPFLAGS="-I${prefix}/include"
+CXXFLAGS="-I${prefix}/include"
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release .
+cmake --build .
+cmake --install .
+"""
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+function platform_filer(p)
+    return contains(p.tags["arch"], "64") && (!haskey(p.tags, "libc") || p.tags["libc"] != "musl") && (!Sys.isbsd(p) || Sys.isapple(p))
```

Comment:
Please don't use p.tags like this, and in particular don't try to parse the `arch` value -- instead use the appropriate tester functions like `nbits(p) == 64`. You can find plenty examples by grepping around in other recipes

Reply:
Agreed, this wasn't the best approach. Instead, I have just listed the platforms explicitly, as this is much easier to read.